.github/workflows/test.yml

```diff
@@ -0,0 +1,66 @@
+name: Scala tests with coverage
+
+on:
+  pull_request:
+    paths-ignore: ['**.md']
+  push:
+    paths-ignore: ['**.md']
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    services:
+      mysql:
+        image: mysql/mysql-server:5.6
+        env:
+          MYSQL_ROOT_PASSWORD: rawls-test
+          MYSQL_USER: rawls-test
+          MYSQL_PASSWORD: rawls-test
+          MYSQL_DATABASE: testdb
+        options: >-
+          --health-cmd="mysqladmin ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 3307:3306
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      # coursier cache action caches both coursier and sbt caches
+      - name: coursier-cache-action
+        uses: coursier/cache-action@v5
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Git secrets setup
+        run: |
+          git clone https://github.com/awslabs/git-secrets.git ~/git-secrets
+          cd ~/git-secrets
+          git checkout ad82d68ee924906a0401dfd48de5057731a9bc84
+          sudo make install
+
+      - name: Secrets check
+        run: |
+          ./minnie-kenny.sh --force
+          git secrets --scan-history
+
+      - name: Run tests
+        run: >-
+          sbt coverage test
```

Comment:
this removes the `clean`  step. I realize it's a fresh checkout so it *_should_* be clean already, but I'd opt to put that step back in. If it truly is already clean, this shouldn't add much to the execution time.